ouroboros-network/src/Ouroboros/Network/NodeToNode.hs

```diff
@@ -95,3 +111,32 @@ nodeToNodeCodecCBORTerm = CodecCBORTerm {encodeTerm, decodeTerm}
       decodeTerm (CBOR.TInt x) | x >= 0 && x <= 0xffff = Right (NodeToNodeVersionData $ fromIntegral x)
                                | otherwise             = Left $ T.pack $ "networkMagic out of bound: " <> show x
       decodeTerm t             = Left $ T.pack $ "unknown encoding: " ++ show t
+
+
+-- |
+-- A specialised version of @'Ouroboros.Network.Socket.connectTo'@.
```

Comment:
Can I ever disuade you from this redundant haddock markup style? :-)

Reply:
Hey, but at least it has the trailing one :).